snapd-glib/requests/snapd-json.c

```diff
@@ -567,6 +586,27 @@ _snapd_json_parse_change (JsonNode *node, GError **error)
     g_autoptr(GDateTime) main_spawn_time = _snapd_json_get_date_time (object, "spawn-time");
     g_autoptr(GDateTime) main_ready_time = _snapd_json_get_date_time (object, "ready-time");
 
+    SnapdChangeData *data = NULL;
```

Comment:
This needs to be `g_autoptr(SnapdChangeData)`

Reply:
Fixed.